src/utilities/sortIntrospection.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {
+  IntrospectionQuery,
+  IntrospectionType,
+  IntrospectionField,
+  IntrospectionDirective,
+} from './introspectionQuery';
+
+/**
+ * Sort the result of Introspection Query.
+ */
+export function sortIntrospectionQuery(
+  introspection: IntrospectionQuery,
+): IntrospectionQuery {
+  const schema = introspection.__schema;
+  return {
+    __schema: {
+      queryType: schema.queryType,
+      mutationType: schema.mutationType,
+      subscriptionType: schema.subscriptionType,
+      types: sortByName(schema.types).map(sortType),
```

Comment:
similar to below, given `sortByName` now takes in a mutable array, we could save one copy here vs. the current code if we flipped this:
```
types: sortByName(schema.types.map(sortType)),
```